stubs/api.php

```diff
@@ -39,6 +39,18 @@
     'namespace' => null,
     'by-resource' => true,
 
+    /*
+    |--------------------------------------------------------------------------
+    | Model Namespace
+    |--------------------------------------------------------------------------
+    |
+    | Here you can decide where your api models live.
+    |
+    | By default Models live on the root of the application like App\Post hence the model-namespace: 'App'
```

Comment:
Formatting of comments - can you ensure the comment line is no longer than the `-----` lines above?

Reply:
done